src/ui/resource/loader.js

```diff
@@ -156,8 +165,15 @@ var Loader = Class(Emitter, function () {
         }
       }
 
+      var audioMap = this._audioMap;
+      var audioToLoad = {};
+      for (var uri in audioMap) {
+        if (uri.indexOf(pathPrefix) == 0) {
```

Comment:
Style fix: always use triple equals; update the block above to use it as well - in general if you see something off-style or questionable in the engine, feel free to fix it.